content/pt/docs/tutorials/kubernetes-basics/deploy-app/deploy-intro.html

```diff
@@ -0,0 +1,115 @@
+---
+title: Usando kubectl para criar uma implantação
+weight: 10
+---
+
+<!DOCTYPE html>
+
+<html lang="en">
+
+<body>
+
+    <link href="/docs/tutorials/kubernetes-basics/public/css/styles.css" rel="stylesheet">
+
+<div class="layout" id="top">
+
+    <main class="content">
+
+        <div class="row">
+
+         <div class="col-md-8">
+          <h3>Objetivos</h3>
+                <ul>
+                    <li> Saiba mais sobre implantações de aplicativos. </li>
+                    <li> Implante seu primeiro aplicativo no Kubernetes com o kubectl. </li>
+                </ul>
+            </div>
+
+            <div class="col-md-8">
+                <h3>Implantações do Kubernetes</h3>
+                <p>
+                Assim que o seu cluster do Kubernetes estiver em execução você pode implementar seu aplicativo em contêiners nele.
+                Para fazer isso, você precisa criar uma configuração do tipo <b> Deployment </b> do Kubernetes. O Deployment define como criar e 
+                atualizar instâncias do seu aplicativo. Depois de criar um Deployment, o Master do Kubernetes
```

Comment:
Here, I believe you need to explain that the master node schedules the creation of the pod. I'm not sure if we should keep Master Node in English or say Nó Master or even Control Plane.

Reply:
Yep, `Nó Master` looks good to me